src/app/components/Timeline/Timeline.tsx

```diff
@@ -54,7 +54,7 @@ export default class Timeline<TItem, TGroup, TOwnProps={}> extends React.PureCom
             return groupRenderer(group);
         } else {
             return (
-                <div>{title}</div>
+                <div style={{fontSize:16}}>{title}</div>
```

Comment:
Might be able to do this in the `DailyTimeline.css` instead.  You'll have to inspect the elements to figure out what css selector you'll need to use.